src/services/user.service.js

```diff
@@ -12,3 +15,33 @@ function getDetails() {
     };
     return fetch(`${config.commonConfig.$apiUrl}/users`, requestOptions).then(handleResponse);
 }
+function setting(username, profilePicture, bio, password1, password2) {
+    const registerBody = {
+        profilePicture : profilePicture,
+        username: username,
+        bio: bio,
+        password1: password1,
+        password2: password2
+    }
+    
+    const existingUsername = JSON.parse(localStorage.getItem('user')).profile.user.username;
+    const url =  stringFormat(`${config.commonConfig.$apiUrl}/${config.userConfig.api.settings.endpoint}`, existingUsername);
+    console.log(url);
+    console.log(registerBody);
+
+
+    return axios.put(
+        url,
+        registerBody
+    )
+        .then((response) => {
+            const data = response.data
+            if (data.profile) {
+                localStorage.setItem('user', JSON.stringify(data));
+                return data.profile
+            }
+        })
+        .catch((error) => {
+            console.log(error);
```

Comment:
Please handle 4XX cases and 5XX cases.

Reply:
the password and bioSize one?

Reply:
Handle HTTP 4XX and 5XX errors when it reaches in error case.

Reply:
Please refer to `catch`-es in `auth.service.js`